calculator/src/components/Button.js

```diff
@@ -0,0 +1,149 @@
+import { Component } from "react";
+
+
+class Button extends Component{
```

Comment:
Great job overall Hector! Just a couple notes. I would say "Button" is not an appropriate name for this component as it's actually, many buttons. All of the buttons actually. I think a better name would be something as simple as "Buttons" plural, or "AllButtons".